### DISCUSSION TITLE
[Weekly Curation Report]: June 26 - 30

### DISCUSSION THREAD

**Discussion by jmillanacosta** [wikipathways/wikipathways-help#44] [curation-reports]
### Recent changes

Recent Changes: 
- Approved WP5372 (@danidi )

### Under construction

Under Construction: 
- WP5355 Added ontology term metabolic disease pathway
- WP5362 Changed legend (nodes to graphical objects
- WP5367 added tag: needs work (description, fix Bibliography
- 

### New pathways

New pathways: 

### Deleted Pathways

Deleted pathways: 

### Additional notes

- Why is WP5342 a test pathway? (more than a month old, seems to be good)
- WP5367 There seems to be a problem fetching/displaying paper information. Tried to query with the PMID from PathVisio and it still couldn't render the paper information

### Update Approved pathway tag

- [X] All Approved pathways tags were updated!

**Comment by khanspers** [comment]
Thanks for the report @jmillanacosta ! 

- [WP5342](https://classic.wikipathways.org/index.php/Pathway:WP5342): I heard back from the original author, who started the pathway during a course. He will do some minor updates to the pathway (updating unlabeled notes etc) to get the pathway approved. I have removed the test pathway tag.
- [WP5367](https://classic.wikipathways.org/index.php/Pathway:WP5367): Not sure why the references don't render on the classic site, and not sure if it would be a problem on the main site. I added a couple of ontology tags.

**Reply by jmillanacosta** [reply]
Thank you, Kristina.
